### DISCUSSION TITLE
科學證實讀小說有益心理健康

### DISCUSSION THREAD

**Discussion by aa103967622** [1101-GS4719-C/Python-Community-GS4719-C#1450] [General]
我最近讀過一篇文章講述到讀小說對於心理健康的益處，我認為閱讀小說可以確實帶來許多好處。

1.閱讀小說能夠讓我們扮演其中的角色(把自己代入角色體驗故事超刺激的)，以不同的方式思考和感受故事中發生的事情。
2.閱讀小說可以讓我們感覺自己成為故事世界的一部分(跟主人公一起建立許多姻緣)，並得到與現實社交相同的滿足感和幸福感。
3.閱讀小說還有助於保持大腦活躍，防止認知能力下降，閱讀可以釋放壓力，提供暫時逃離現實的避難所(沉浸在小說的世界裡也不錯)。

文章來源：https://dq.yam.com/post/9728

最後想知道大家最近有什麼推薦的小說嗎？什麼類型都可以！

**Comment by Alexgns** [comment]
《紅皇后》是由美國作家維多利亞·阿維亞爾（Victoria Aveyard）創作的一部暢銷小說，是《紅皇后系列》的首部作品
個人覺得還不錯，如果你喜歡奇幻冒險小說，探索權力、社會不公和個人勇氣的主題，那麼《紅皇后》可能是一本值得一讀的小說。它提供了一個充滿魅力和驚喜的故事世界，讓讀者沉浸其中，隨著主人公的冒險一同成長。

**Comment by ALAN30359704** [comment]
我認為看小說，的確有助於大腦的活化，但是在現在這個時代，我認為看小說對於來說是件非常痛苦的事情，原因是小說只有文字，但是沒有圖就讓人覺得相較於漫畫而言閱讀起來相對的吃力也可以換個角度來思考我們說一張圖勝過千言萬語，所以我們人本身對圖案的理解能力應該是較高的！

**Comment by Elaine2899** [comment]
我自己非常喜歡看小說，雖然事情越來越多也讓我越來越沒有時間可以讀課外書，但我還是對自己有期許：平日趕工把事情做完，讓假日有時間可以有一些閒暇可以多讀一點小說。在看小說時，我感覺會把自己剝離現實生活，投入虛擬的世界裡。在某些段落，甚至可以在腦海裡投影出文字描述的畫面，當畫面出現時，這些故事就會進入我的長期記憶區，讓我在某些相似的時刻想到小說裡的這個段落這個畫面，重拾當時看小說時的感動。因此我覺得看小說不僅可以在當下放鬆，也可以是一輩子的感動。

**Comment by Aiame** [comment]
在我上高中前，從來就不覺得看小說好玩，但在老師推薦看英文小說之後，就從此愛上了，而我常常都是在睡前看，當作放鬆。直到有一天聽到老師說睡前看有助於記憶，覺得很神奇，又看到有國外大學研究正向或獎賞對記憶有幫助，自己體感好像有變好，不知道是不是心理作用就是了，或許是看一看就睡著，把他當作是獎勵機制嗎?!~。中文小說我不清楚，但我覺得英文小說的帶入感，真的蠻好的，
我自己是最推薦看飢餓遊戲。https://www.inc.com/minda-zetlin/memory-long-term-retention-neuroscience-reward-experiment.html (國外研究

**Comment by Evening218** [comment]
我個人非常喜歡看小說,而且甚至是從恐怖驚悚題材開始的,但對我來說,其實有內容的小說非常有幫助,甚至我在高中時期寫作文的時候,都是從看過的小說裡面找想法跟用詞,而且只有文字的小說還可以自行想像畫面,真的很推薦
